components/App/index.js

```diff
@@ -0,0 +1,47 @@
+import React, { Component } from 'react'
+import { StyleSheet, View } from 'react-native'
+import PropTypes from 'prop-types'
+
+import Board from '../Board'
+import Button from '../Button'
+
+const BOARDS = [{name: 'Bootcamp'}];
+const USERS = {name: 'Admin'};
+
+export default class App extends Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            boards: BOARDS,
+            users: USERS,
+            boardInput: ''
+        }
+    }
+
+    createBoard() {
+        this.setState(s => {return {boards: [...s.boards, {name: s.boardInput}]}});
```

Comment:
more readable to do `this.setState(s => ({boards: [...s.boards, {name: s.boardInput}]}));`